apps/utils/locust_tests/common/bfd_user_base.py

```diff
@@ -35,6 +36,17 @@ def _(environment: Environment, **kwargs) -> None:
 
     validation.setup_failsafe_event(environment)
 
+    test_host: str = environment.parsed_options.host
+    if test_host.endswith(("/", "\\")):
+        logger = logging.getLogger()
+        logger.error(
+            "--host must not end with any trailing slashes. "
+            "Please remove the slashes and try again."
+        )
```

Comment:
I definitely agree with this -- I _had_ tried to do so by modifying `environment.parsed_options.host` in the `init` event handler but had no success. _However_, at the time I didn't realize that there is a `host` field on the `environment` object that is properly mutable. 

I've updated the PR to **remove** the trailing slashes rather than warn and exit, and I've also updated the new documentation to reflect that trailing slashes are removed.